internal/agent/device/systeminfo/network.go

```diff
@@ -307,14 +315,22 @@ func getDefaultRouteIPv6(reader fileio.Reader) (*DefaultRoute, error) {
 		// check if this is the default route (destination ::/0, which is 00000000000000000000000000000000 with prefix 00)
 		// https://mirrors.deepspace6.net/Linux+IPv6-HOWTO/proc-net.html
 		if fields[0] == "00000000000000000000000000000000" && fields[1] == "00" {
+			interfaceName := fields[9]
```

Comment:
just making sure we are len checking before somewhere.

Reply:
Len checking the size of the name? We aren't doing that. Len checking to ensure we have at least 10 fields? That happens a couple of lines up.